lib/beaker/host.rb

```diff
@@ -202,7 +202,13 @@ def ip
     #Examine the host system to determine the architecture
     #@return [Boolean] true if x86_64, false otherwise
     def determine_if_x86_64
```

Comment:
Hm, taking a second look over this, this is why we have the object inheritance structure that we do.  This could be divided up by having a custom determine_if_x86_64 in the pswindows/exec hosts and then a default method in hosts.rb.  That way all the custom ps windows work is in a single location.